app.js

```diff
@@ -120,7 +120,7 @@ app.use(
                 "'self'",
                 "blob:",
                 "data:",
-                "https://res.cloudinary.com/dnbxvhjte/",
+                "https://res.cloudinary.com/"+ process.env.CLOUDINARY_CLOUD_NAME +"/",

```

Comment:
this now doesn't have your hardcoded CLOUDINARY_CLOUD_NAME

anyone can add their cloudinary cloud here and see images